_studio/shared/include/mfx_config.h

```diff
@@ -166,6 +166,10 @@
         #undef MFX_ENABLE_HEVC_VIDEO_FEI_ENCODE
         #undef MFX_ENABLE_VP8_VIDEO_DECODE_HW
     #endif // #if defined(AS_HEVCD_PLUGIN)
+    #if defined(AS_CAMERA_PLUGIN)
```

Comment:
I don't think we have camera plugin in open source. If so, these change in config is useless. We don't build it and thus we can't support it. Please, drop the change.